docs/standard/linq/functional-construction-linq-to-xml.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Functional Construction - .NET"
+ms.date: 12/01/2019
+dev_langs:
+  - "csharp"
+  - "vb"
+ms.assetid: 57a82bcf-de03-4f1c-a0c8-9a76e989d542
+---
+# Functional Construction (C# and Visual Basic)
```

Comment:
One thing that will probably help you in this task and that we're trying to restore here is that these topics used to be unified before. So here's this one before: https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2012/bb387019(v%3dvs.110)
We can't simply just copy and paste because some might have changed over time but it would give you a good frame.

So we should make it look like that.